src/app/models/welding_point.py

```diff
@@ -13,6 +13,9 @@ class WeldingPoint(Base):
     x = Column(Float)
     y = Column(Float)
     z = Column(Float)
+    x_original = Column(Float)
+    y_original = Column(Float)
+    z_original = Column(Float)
```

Comment:
Could we already add a `nullable=false` here? Did it the first time [here](https://github.com/SQE22-RoboticWelding/model-transformer-welding-backend/blob/da011fa011031b8f06a22d6711609d90237d21c2/src/app/models/generation_template.py#L12), and we should have done that everywhere where necessary, IMHO.

Reply:
Addressed